pkg/distro/defs/rhel-10/distro.yaml

```diff
@@ -94,14 +92,16 @@
     dnf_config:
       set_release_ver_var: true
 
-  sap_pkgset: &sap_pkgset
+  sap_image_config: &sap_image_config
+    <<: *sapapps_image_config
+    tuned:
+      profiles: ["sap-hana"]
+
+  sap_base_pkgset: &sap_base_pkgset
     include:
       # RHBZ#2076763
       - "@Server"
-      # SAP System Roles
-      # https:#access.redhat.com/sites/default/files/attachments/rhel_system_roles_for_sap_1.pdf
       - "ansible-core"
-      - "rhel-system-roles-sap"
```

Comment:
Nitpick: `ansible-core` is there IMHO only due to the RHEL system roles, which are Ansible roles. Not sure how useful it is for the SAP Apps image...

Reply:
I realised this from the grouping under the comment, but decided to do the bare minimum change in the end.  Since you brought it up too, I think I'm going to open a follow-up PR and move `ansible-core` to the extras as well.